http-clients/feign-common/src/test/java/org/triplea/http/client/AuthenticationHeadersTest.java

```diff
@@ -0,0 +1,27 @@
+package org.triplea.http.client;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.HashMap;
+import org.junit.jupiter.api.Test;
+import org.triplea.domain.data.ApiKey;
+
+class AuthenticationHeadersTest {
+  final ApiKey apiKey = ApiKey.newKey();
+  final AuthenticationHeaders authenticationHeaders = new AuthenticationHeaders(apiKey);
+
+  @Test
+  void createHeaders() {
+    assertTrue(authenticationHeaders.createHeaders() instanceof HashMap);
```

Comment:
Why is it important that the return value be a `HashMap` and not say a `Map`? If the other test assertions managed to pass and this one failed, would the code be incorrect?


Nit:
- conventionally we use hamcrest assertions. IIRC there is an 'instanceof' check (If think `IsInstanceOf.instanceOf`) that can be used to make this nicer. A first priority is often to get tests in place, a next prioirty is to make it easy to debug failing tests (this makes the next developers life much easier and avoids the situation where small changes wind up spending hours fixing tests). So.. using the hamcrest match would help becuase it would state the type of instance that was found vs expected, vs here we would have just a simple yes/no and no further information if it were to fail.

Reply:
Cool. I'll switch that over to hamcrest. 

I mess that up regarding the`HashMap`. I saw the `new HashMap<>()` in the method that was in my mind. I'll switch it to `Map` because that is what the method says it will return.